lightning/src/routing/router.rs

```diff
@@ -519,6 +519,11 @@ impl Path {
 			None => self.hops.last().map(|hop| hop.cltv_expiry_delta)
 		}
 	}
+
+	/// True if this [`Path`] has at least one Trampoline hop.
+	pub fn has_trampoline_hops(&self) -> bool {
```

Comment:
This one can be combined with the next commit. Otherwise I have to go back and forth as a reviewer anyway.